sse-event-handler/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
remove /idea directory from committing. Remove those files and add .idea to gitignore file https://github.com/wso2-extensions/identity-outbound-auth-email-otp/blob/master/.gitignore#L9